php/sync-with-algolia/src/utils.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+$staticFolder = dirname(__FILE__) . '/../static';
+
+/**
+ * Returns the contents of a file in the static folder
+ * @param string $fileName
+ * @return string Contents of static/{$fileName}
+ */
+function get_static_file(string $fileName): string
```

Comment:
notice `get_static_file` and `fileName`. Is this common pattern in PHP? (underscore_case, and camelCase)